types/meteor/mongo.d.ts

```diff
@@ -160,6 +160,8 @@ declare module 'meteor/mongo' {
             reactive?: boolean | undefined;
             /**  Overrides `transform` on the  [`Collection`](#collections) for this cursor.  Pass `null` to disable transformation. */
             transform?: Transform<T> | undefined;
+            /** specifies the index */
+            hint?: string | undefined;
```

Comment:
According to [the docs](https://docs.meteor.com/api/collections.html#Mongo-Collection-find), `hint` is also allowed to be a `Record<string, number>`

Reply:
@ebroder do you think `hint?: string | Partial<Record<keyof T, number>> | undefined;` will work?

Reply:
I'd go with the exact same approach as the `mongodb` package itself:
1. https://github.com/mongodb/node-mongodb-native/blob/25d22b2619158e8555863f8505fdafa7f2f0d5da/src/operations/operation.ts#L18
2. https://github.com/mongodb/js-bson/blob/35bb2a279343439e1306a0eeb8d65d1abf910929/src/bson.ts#L105-L108

Reply:
Okay, then forget about auto-complete benefit. This will works as mongodb hint option:
```
/** (Server only) Overrides MongoDB's default index selection and query optimization process. Specify an index to force its use, either by its name or index specification. */
hint?: string | Record<string, any> | undefined;
```